release-notes/aws/v9.0.1.md

```diff
@@ -0,0 +1,25 @@
+## :zap: Giant Swarm Release 9.0.1 for AWS is now active for you! :zap:
+
+This release includes some reliability and user experience improvements.
+
+Below, you can find more details on components that were changed with this release.
+
+### chart-operator [v0.12.0](https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0120-2020-03-09)
+
+- Added chartmigration resource for migrating from chartconfig to chart CRs.
+- Updated release resource.
+  - Do not wait when installing or updating long running Helm releases.
+  - Use version field from chart CR to reduce number of HTTP requests to pull chart tarballs.
+  - Wait for deleted Helm release before removing finalizer. 
+- Updated status resource.
+  - Improve reason field in CR status when installing a chart fails.
+- Removed legacy chartconfig controller.
+
+### cluster-operator [v0.23.4](https://github.com/giantswarm/cluster-operator/releases/tag/v0.23.4)
+
+- Stop setting IC replicas count.
+- Re-enable proxy protocol by default for AWS clusters.
+- Fix cluster deletion by gracefully handling Tenant Cluster API errors.
+- Updated to nginx IC App v1.3.0 with:
```

Comment:
Release definition includes update to nginx 1.6.1 with HPA enabled - this is still not covered here; definitely should not be under cluster-operator section.